MidBone/portal
# Ui to merge with payment (#6)

* Payment gateway with ui (#4)

* Ui (#3)

* specification_unit change 0

* Product price

* small changes

* product details added

* product details page implemented

* footer removed from index page and some css changes

* pages added wish list and contact us

* cart.jsp added

* wish list page changes and prodct page changes

* Add to cart page is implemented and other bug fixes and enhancements done

* modified import.sql for product_offer table

* test folder is deleted

* product page complete

* Delete .DS_Store file

* Delete classpath

* final commit for product details page from github

* OTP related

* OTP implementation, get update with latest structure from import.sql

* otp with checkout

* partially committing user control options

* committing stashed changes (#5)

* otp for checkout page complete

* css changes

* image replace

* OTP and checkout page

* OTP and check out page

* import statement is updated
